### PR TITLE
fix(getting-started): remove some code snippets, re-up local testing

### PR DESCRIPTION
**Pull request by bodinsamuel**
## Describe your changes

Fixes NAN-469

- Remove unnecessary code snippets
- Reup the possibility to access getting started locally with a global state

### CODE REVIEW

File: packages/webapp/src/App.tsx [hunk 1]
```diff
@@ -55,6 +55,12 @@ const App = () => {
     const SentryRoutes = Sentry.withSentryReactRouterV6Routing(Routes);
     const env = useStore((state) => state.cookieValue);
     const signout = useSignout();
+    const setShowGettingStarted = useStore((state) => state.setShowGettingStarted);
+    const showGettingStarted = useStore((state) => state.showGettingStarted);
+
+    useEffect(() => {
+        setShowGettingStarted(env === 'dev' && (isCloud() || isLocal()));
```
Comment (khaliqgant) [inline packages/webapp/src/App.tsx:62]: why a useeffect here? What issue were you seeing?

**Review by khaliqgant — CHANGES_REQUESTED** [review]
Question about useEffect inline

File: packages/webapp/src/App.tsx (hunk 1 above)
Comment (bodinsamuel) [reply packages/webapp/src/App.tsx:62]: It's just to setup the global state so we don't have to rewrite the condition multiple times, and keep it in sync with environment change. would you have done something differently?

Comment (khaliqgant) [reply packages/webapp/src/App.tsx:62]: ah ok gotcha -- to keep things DRY. Woks for me.

**Review by khaliqgant — APPROVED** [review]

File: packages/webapp/src/App.tsx [hunk 2]
```diff
@@ -55,6 +55,12 @@ const App = () => {
     const SentryRoutes = Sentry.withSentryReactRouterV6Routing(Routes);
     const env = useStore((state) => state.cookieValue);
     const signout = useSignout();
+    const setShowGettingStarted = useStore((state) => state.setShowGettingStarted);
+    const showGettingStarted = useStore((state) => state.showGettingStarted);
+
+    useEffect(() => {
```
Comment (TBonnin) [inline packages/webapp/src/App.tsx:61]: should we use `useState` instead?
```
const [showGettingStarted, setShowGettingStarted] = useState(env === 'dev' && (isCloud() || isLocal()));
```

Comment (bodinsamuel) [reply packages/webapp/src/App.tsx:61]: The same condition is reused in an other file, so it's less ideal

**Pull request merged**